Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -139,15 +139,16 @@ def get_test_playbooks_configured_with_integration(self, integration_id):
 ID_SET = {}
 CONF: TestConf = None  # type: ignore[assignment]
 
```

Comment:
Can we maybe perform all the `os.path.join(ARTIFACTS_FOLDER, <file-name.ext>)` as assignments to variables before the `if-elif-else` block conditions? Nesting it inside the call to `open` impinges on the readability a bit IMO plus we are repeating the ops twice here, once in the conditional expression and again in the call to `open` the file.